server/BankServer/BankServer/Models/ChangePinModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BankServer.Models
+{
+    public class ChangePinModel
+    {
+        public string CardNum { get; set; }
+        public string OldPin { get; set; }
+        public string NewPin { get; set; }
```

Comment:
I suppose this won't work with current naming. Can you please provide a screenshot from PostMan? We need to accept `cardNum` with lowercase starting letter.